src/graphorm/index.ts

```diff
@@ -412,9 +439,8 @@ const obj = new GraphORM({
         alias: { field: 'url', type: 'string' },
         transform: (value: string): string => domainOnly(value),
       },
-      title: createI18nField({
+      title: createSmartTitleField({
```

Comment:
This will return only as titlte right?
I think we need the addition of `fieldAs` here maybe?

Output should always include original title still as well.

Reply:
I am not sure as well

Reply:
We currently only return `title` even when they are translated.
During RFC meeting, we decided on doing it like translation, and replacing it in title.

So now when you have clickbait shield enabled, it will attempt to return the alternative title first, then falls back to translated title, and finally original title.

Reply:
Ah but then we also said for OG/SEO purposes we should always have the original as well right?
Also when translating back from this it would be easier if it's just there.

My suggestion is we keep title as is, but have `altTitle` as secondary field.
OR title is always rendered title (what you do above), but we add originalTitle somewhere? (and modify only for SEO etc)?

Reply:
For OG/SEO, it's accessed as anonymous user, so it will fallback to original title

Reply:
As discussed, this is fine as OG/SEO is fetched as anon user, so it always returns original title.